README-UI.md

```diff
@@ -0,0 +1,183 @@
+# Centralized UI Workflows
+
+- [Methodology](#methodology)
+- [Versioning](#versioning)
+- [Using in your repository](#using-in-your-repository)
+- [Configuration](#configuration)
+  - [Node/Yarn configuration](#nodeyarn-configuration)
+  - [Linting](#linting)
+  - [Unit tests](#unit-tests)
+    - [Jest](#jest)
+    - [Bigtest](#bigtest)
+  - [Sonarcloud](#sonarcloud)
+  - [Translations](#translations)
+  - [Module descriptor generation](#module-descriptor-generation)
+  - [Module publication](#module-publication)
+- [Secrets](#secrets)
+
+To improve the consistency and reliability of the CI/CD workflows for the FOLIO front-end repositories, we are migrating to a centralized workflow. This enables more frequent and centralized updates, encourages shared standards, and reduces the maintenance burden on individual repositories.
+
+## Methodology
+
+By separating each step into its own job, we can run them in parallel, which should speed up the overall process. This also allows us to easily add new steps in the future, and to run only the steps that are necessary for a given change.
+
+Here are all of the individual jobs and how they interact:
+
+![UML Diagram](./ui-diagram.png)
+
+## Versioning
+
+TODO: handle versioning of central workflow with tags? branches? semver? flower names? R#-20##?
```

Comment:
In the workflow files I see statements like 
```
uses: actions/checkout@v4
```
Let's do the same thing. It seems to be Standard Operating Procedure. If we follow that, then we're also obeying Principle Of Least Surprise. Win, win.